### DISCUSSION TITLE
Data Hub - Fetch a Car Product - Query Schema

### DISCUSSION THREAD

**Discussion by triongroup** [pimcore/pimcore#9240] [Q&A / Support] [answered]
Hello everyone!
During my study I found en error while tried to execute example query from documentation via Data Hub IExplorer on Demo site (https://demo.pimcore.fun/admin/).
<img width="1304" alt="Снимок экрана 2021-05-26 в 17 45 34" src="https://user-images.githubusercontent.com/13614676/119680921-4a662e80-be4a-11eb-971a-c84dc503acd5.png">
<img width="1297" alt="Снимок экрана 2021-05-26 в 17 45 50" src="https://user-images.githubusercontent.com/13614676/119680932-4c2ff200-be4a-11eb-8eb2-e86316644135.png">
all was done in according to tutorial video and documentation.
But I got en error. What I do wrong? Or may be it is a bug?

**Comment by weisswurstkanone** [comment, marked as answer]
Already fixed via https://github.com/pimcore/data-hub/commit/8ddf619b937917a4410927d4ebb25f2e22e65e41
Will tag a new release now.
